docs/data-types/sorted-sets.md

```diff
@@ -29,27 +29,16 @@ represent sorted sets). They are ordered according to the following rule:
 * If B and A are two elements with a different score, then A > B if A.score is > B.score.
 * If B and A have exactly the same score, then A > B if the A string is lexicographically greater than the B string. B and A strings can't be equal since sorted sets only have unique elements.
 
-Let's start with a simple example, adding a few selected hackers names as
-sorted set elements, with their year of birth as "score".
-
-    > zadd hackers 1940 "Alan Kay"
-    (integer) 1
-    > zadd hackers 1957 "Sophie Wilson"
-    (integer) 1
-    > zadd hackers 1953 "Richard Stallman"
-    (integer) 1
-    > zadd hackers 1949 "Anita Borg"
-    (integer) 1
-    > zadd hackers 1965 "Yukihiro Matsumoto"
-    (integer) 1
-    > zadd hackers 1914 "Hedy Lamarr"
-    (integer) 1
-    > zadd hackers 1916 "Claude Shannon"
-    (integer) 1
-    > zadd hackers 1969 "Linus Torvalds"
-    (integer) 1
-    > zadd hackers 1912 "Alan Turing"
-    (integer) 1
+Let's start with a simple example, we'll add all our racers and the score they got in the first race:
+
+{{< clients-example ss_tutorial zadd >}}
+> ZADD racer_scores 10 "Norem"
+(integer) 1
+> ZADD racer_scores 12 "Castilla"
+(integer) 1
+> ZADD racer_scores 8 "Sam-Bodden" 10 "Royce" 6 "Ford" 14 "Prickett"
+(integer) 4
+{{ /clients-example> }}
```

Comment:
The tags needs to be fully enclosed in < >. Looks like the there's the same issue all the others below.

```suggestion
{{ < /clients-example > }}
```